code_quality.gemspec

```diff
@@ -23,10 +23,10 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "bundler-audit"
   spec.add_dependency "brakeman"
-  spec.add_dependency "rubycritic", "~> 3.5.0"
-  spec.add_dependency "rubocop", "~> 0.59.0"
-  spec.add_dependency "rubocop-github", "~> 0.12.0"
-  spec.add_dependency "code_metric_fu", "~> 4.14.4"
+  spec.add_dependency "rubycritic", ">= 3.5.0"
```

Comment:
should never use `>=` restriction  in a gem's gemspec since the new version of dependency may have breaking changes